mm2src/common/log.rs

```diff
@@ -837,7 +821,8 @@ impl LogState {
                 val: t.val(),
             })
             .collect();
-        for en in self.tail.spinlock(77).unwrap().iter() {
+        let tail = self.tail.lock();
+        for en in tail.iter() {
```

Comment:
`for en in self.tail.lock().iter() {`

Reply:
To be honest, I thought that compiler should fail due to `MutexGuard` is dropped when `.iter()` is called taking into account that `.iter()` takes inner value by reference. 🤔

Reply:
As we discussed before, it's better to leave it as it is to make it more obvious that the `MutexGuard` still exists.